### ISSUE TITLE
link text for footnotes is wrong (using compiler _remark_)

### ISSUE THREAD

**Issue by salim-b** [simov/markdown-viewer#71]
Consider the following example Markdown document

```
# H1

blabla[^footnote-id]

[^footnote-id]: super-important footnote content.
```

which is rendered to (using _remark_ as the compiler):

![test md](https://user-images.githubusercontent.com/20040931/49596083-c9fd9b80-f979-11e8-985e-0a4cd5d1797d.png)

Obviously, the superscript link text to the footnote after "blabla" should be the number `1` and not `footnote-id`. I don't know if this is a general problem with _remark_ or specific to _markdown-viewer_.

**Comment by simov** [comment]
It is a long standing issue with with remark, initially reported here https://github.com/simov/markdown-viewer/issues/35 Last time I checked it wasn't fixed, so I'll take a look at it during the next release cycle.

**Comment by ftrebien** [comment]
Maybe this could be a solution for Remark: https://www.npmjs.com/package/remark-numbered-footnotes

The numbering works fine with _markdown-it-footnote_ in Visual Studio Code's [Markdown Footnotes](https://github.com/mjbvz/vscode-markdown-footnotes) extension.

**Comment by simov** [comment]
Seems to be fixed in Markdown Viewer v5.2 for both of the compilers that has support for footnotes: `remark` and `markdown-it`.
